packages/language/test/fourslash/preprocessor/include/incafter-in-config.ts

```diff
@@ -33,8 +33,7 @@
 
 // @filename: main.pli
 ////*PROCESS;
-//// DCL TEST FIXED;
 
-preprocessor.expectTokens(`*PROCESS;
-  DCL TEST FIXED;
+preprocessor.expectTokens(`*PROCESS INCAFTER();
```

Comment:
Question: Why do we need the `INCAFTER()` here? I don't think it should be required.

Reply:
It is for the moment, but I initially thought the same as well. The updated process group in the test is also impacting the resulting token stream here, so we're getting more tokens than expected. I chatted with @ssmifi about this yesterday, he said he was going to take a look there. I think we all agree, this part should be independent from any config changes we apply to the tests.

Beyond that using the `INCAFTER()` options works now, and will also work as expected after we address this config issue.

Reply:
Forgot to update here, with the new changes from @ssmifi we don't need to invoke this directly for the test's sake.